### ISSUE TITLE
Classifier free guidance in i2vgen-xl config

### ISSUE THREAD

**Issue by jlia904** [ali-vilab/i2vgen-xl#63]
Hi,

I was looking at the i2vgen_xl_train.yaml for the i2vgen-xl model and noticed that `p_zero` for classifier-free guidance is set to zero, is this supposed to be set to zero in your training setup?

Thanks!

**Comment by Steven-SWZhang** [comment]
Hi, during the training stage, you can set `p_zero=0.1`. In fact, we set the dropout rate in the model forward function.
